src/topology/path_connected.lean

```diff
@@ -372,6 +401,13 @@ begin
     simp [hst, mul_inv_cancel (@two_ne_zero ℝ _ _)] }
 end
 
+lemma continuous_trans {x y z : X} : continuous (λ ρ : path x y × path y z, ρ.1.trans ρ.2) :=
```

Comment:
Should we also have a `@[continuity]` lemma `continuous f → continuous g → continuous (λ x, (f x).trans (g x))` named `_root_.continuous.path_trans`?

Reply:
Isn't it a special case of `trans_continuous_family` above?

Reply:
That lemma is about coercions to functions, not about `path` objects.

Reply:
Oh, I see. Is the lemma below what you had in mind, then?

Reply:
You can prove this using the next lemma as `continuous_fst.path_trans continuous_snd`.